content/integrations/azure-keyvault.md

```diff
@@ -246,20 +294,164 @@ As an external application, you must register KES in [Azure Active Directory](ht
+| `address`                     | The network address and port the KES server listens to on startup. Defaults to port `7373` on all host network interfaces. |
+| `root`                        | The identity for the KES superuser (`root`) identity. Clients connecting with a TLS certificate whose hash (`kes identity of client.cert`) matches this value have access to all KES API operations. Specify `disabled` to remove the root identity and rely only on the `policy` configuration for controlling identity and access management to KES. |
+| `tls`                         | The TLS private key and certificate used by KES for establishing TLS-secured communications. Specify the full path for both the private `.key` and public `.cert` to the `key` and `cert` fields, respectively. |
+| `policy`                      | Specify one or more [policies]({{< relref "/tutorials/configuration.md#policy-configuration" >}}) to control access to the KES server. MinIO SSE requires access to the following KES cryptographic APIs: <br><br> `/v1/key/create/*` <br> `/v1/key/generate/*` <br> `/v1/key/decrypt/*` <br><br> Specifying additional keys does not expand MinIO SSE functionality and may violate security best practices around providing unnecessary client access to cryptographic key operations. <br><br> You can restrict the range of key names MinIO can create as part of performing SSE by specifying a prefix before the `*.` For example, `minio-sse-*` only grants access to `create`, `generate`, or `decrypt` keys using the `minio-sse-` prefix. <br><br>KES uses mTLS to authorize connecting clients by comparing the hash of the TLS certificate against the `identities` of each configured policy. Use the `kes identity of` command to compute the identity of the MinIO mTLS certificate and add it to the `policy.<NAME>.identities` array to associate MinIO to the `<NAME>` policy. |
+| `keys`                        | Specify an array of keys which *must* exist on the root KMS for KES to successfully start. KES attempts to create the keys if they do not exist and exits with an error if it fails to create any key. KES does not accept any client requests until it completes validation of all specified keys.|
+| `cache`                       | Specify expiration of cached keys in `#d#h#m#s` format. Unexpired keys may be used in the event the KMS becomes temporarily unavailable. <br><br> Entries may be set for `any` key, `unused` keys, or `offline` keys. <br><br> If not set, KES uses values of `5m` for all keys, `20s` for unused keys, and `0s` for offline keys. |
+| `log`                         | Enable or disable output for `error` and `audit` type logging events to the console. |
+| `keystore.azure.keyvault.endpoint` | The KeyVault endpoint. For example, `https://my-instance.vault.azure.net`. |
+| `keystore.azure.keyvault.credentials.tenant_id` | The ID of the tenant the client belongs to to use to authenticate to KeyVault, that is a UUID. |
```

Comment:
perhaps this shorter one works?
```suggestion
| `keystore.azure.keyvault.credentials.tenant_id` | The ID (UUID) of the tenant the client belongs to, used to authenticate to KeyVault.
```

If so, they all could do `The ID (UUID)`

Reply:
Yeah, taking it for all of them.